ansible/tasks/oauth2-proxy.yml

```
@@ -43,3 +43,28 @@
     name: "oauth2-proxy"
     enabled: true
     state:  "started"
+
+# Same thing but now for admin only oauth access
+
+- name: "copy allowed users file"
+  template:
+    src: "templates/etc/oauth2_proxy/allowed-users"
+    dest: "/etc/oauth2_proxy/allowed-users"
+
+- name: "copy oauth2_proxy admin config"
+  template:
+    src: "templates/etc/oauth2_proxy/admin-config.cfg.j2"
+    dest: "/etc/oauth2_proxy/admin-config.cfg"
+    notify: "restart oauth2-proxy-admin"
```

I believe that this needs to be dedented to have the `notify` work correctly

```suggestion
  notify: "restart oauth2-proxy-admin"
```